### DISCUSSION TITLE
Contributing and i18n

### DISCUSSION THREAD

**Discussion by lucasfcunha** [bensheldon/good_job#1537] [Q&A]
Hi,

I'm currently coding an enhancement to GoodJob, which will include a page where we can examine the big picture of discarded jobs and take action. 

https://github.com/bensheldon/good_job/issues/1464

Here is what the page looks like:![image](https://github.com/user-attachments/assets/5b2d4407-50ed-421c-a00d-9bcb2f273114)

My question is, how do I handle i18n in this case? I can add the English and Portuguese translations, but the rest will not be that easy. What's the process of having those files translated? In the first moment, should I just add those keys to all other languages with the English words in them?

**Comment by bensheldon** [comment]
The simplest thing is to open a PR with only the `en.yml` filled out and I can do the rest. Which looks like: using `i18n-tasks` to generate translations with Google Translate (or sometimes I'll use ChatGPT, though that workflow is a little more annoying).
